src/backend/joanie/core/utils/issuers.py

```diff
@@ -0,0 +1,33 @@
+"""Utils that can be useful throughout Joanie's core app for document issuers"""
+from pathlib import Path
+
+from django.template import Context
+from django.template.engine import Engine
+from django.template.loader import get_template
+
+from weasyprint import CSS, HTML
+from weasyprint.text.fonts import FontConfiguration
+
+
+def generate_document(path: str, context: dict) -> bytes:
```

Comment:
Actually what you are passing is not a path, just a name:
```suggestion
def generate_document(name: str, context: dict) -> bytes:
```